src/OpenTelemetry.ResourceDetectors.AWS/AWSResourcesEventSource.cs

```diff
@@ -8,10 +8,16 @@
 namespace OpenTelemetry.ResourceDetectors.AWS;
 
 [EventSource(Name = "OpenTelemetry-ResourceDetectors-AWS")]
-internal sealed class AWSResourcesEventSource : EventSource
+internal sealed class AWSResourcesEventSource : EventSource, IServerCertificateValidationEventSource
 {
     public static AWSResourcesEventSource Log = new();
 
+    private const int EVENTIDFAILEDTOEXTRACTATTRIBUTES = 1;
```

Comment:
Consider (same for other consts). I think that we follow Pascale Case for most of the cosnts in this repo.

```suggestion
    private const int EventIdFailedToExtractAttributes = 1;
```